From: =?UTF-8?q?Leonard=20G=C3=B6hrs?= <l.goehrs@pengutronix.de>
Date: Mon, 14 Jul 2025 11:06:20 +0200
Subject: [PATCH] Release 6.16/customers/lxa/lxatac/20250714-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index c09766beb7ef..75adaee83bde 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 16
 SUBLEVEL = 0
-EXTRAVERSION = -rc6
+EXTRAVERSION = -rc6-20250714-1
 NAME = Baby Opossum Posse
 
 # *DOCUMENTATION*
