From: =?UTF-8?q?Leonard=20G=C3=B6hrs?= <l.goehrs@pengutronix.de>
Date: Mon, 20 Feb 2023 15:43:13 +0100
Subject: [PATCH] Release 6.2/customers/lxa/lxatac/20230220-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 3f6628780eb2..17aed25ba2a0 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 2
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20230220-1
 NAME = Hurr durr I'ma ninja sloth
 
 # *DOCUMENTATION*
