From: =?UTF-8?q?Leonard=20G=C3=B6hrs?= <l.goehrs@pengutronix.de>
Date: Tue, 27 Jun 2023 10:57:45 +0200
Subject: [PATCH] Release 6.4/customers/lxa/lxatac/20230627-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index e51e4d9174ab..48a3313c9438 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 4
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20230627-1
 NAME = Hurr durr I'ma ninja sloth
 
 # *DOCUMENTATION*
