From: =?UTF-8?q?Leonard=20G=C3=B6hrs?= <l.goehrs@pengutronix.de>
Date: Mon, 8 Jan 2024 07:15:42 +0100
Subject: [PATCH] Release 6.7/customers/lxa/lxatac/20240108-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index c6f549f6a4ae..bef921c1d62b 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 7
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20240108-1
 NAME = Hurr durr I'ma ninja sloth
 
 # *DOCUMENTATION*
