From: =?UTF-8?q?Leonard=20G=C3=B6hrs?= <l.goehrs@pengutronix.de>
Date: Tue, 9 May 2023 14:55:26 +0200
Subject: [PATCH] Release 6.3/customers/lxa/lxatac/20230509-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index f5543eef4f82..23955d8590ec 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 3
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20230509-1
 NAME = Hurr durr I'ma ninja sloth
 
 # *DOCUMENTATION*
