From: =?UTF-8?q?Leonard=20G=C3=B6hrs?= <l.goehrs@pengutronix.de>
Date: Fri, 1 Sep 2023 08:51:51 +0200
Subject: [PATCH] Release 6.5/customers/lxa/lxatac/20230901-2

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 2fdd8b40b7e0..d1ea88678301 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 6
 PATCHLEVEL = 5
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20230901-2
 NAME = Hurr durr I'ma ninja sloth
 
 # *DOCUMENTATION*
