From: Michael Grzeschik <m.grzeschik@pengutronix.de>
Date: Mon, 2 May 2022 14:56:15 +0200
Subject: [PATCH] HACK: Workaround timeout after core reset on some lxatac
 boards

---
 drivers/usb/dwc2/dwc2.c | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/drivers/usb/dwc2/dwc2.c b/drivers/usb/dwc2/dwc2.c
index 1b5981c2d54e..cb57be82bf81 100644
--- a/drivers/usb/dwc2/dwc2.c
+++ b/drivers/usb/dwc2/dwc2.c
@@ -90,6 +90,14 @@ static int dwc2_probe(struct device_d *dev)
 	if (ret)
 		goto error;
 
+	/* FIXME: This is a temprorary workaround where we increase the
+	 * time between enabling the phy and resetting the dwc2 core.
+	 * Without this, some lxatac2 will run into timeout after
+	 * resetting the core, which will lead to non functional usb
+	 * gadget afterwards.
+	 */
+	mdelay(1);
+
 	/*
 	 * Reset before dwc2_get_hwparams() then it could get power-on real
 	 * reset value form registers.
