From: =?UTF-8?q?Leonard=20G=C3=B6hrs?= <l.goehrs@pengutronix.de>
Date: Wed, 18 Sep 2024 14:45:25 +0200
Subject: [PATCH] Release 2024.09.0/customers/lxa/tac/20240918-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 29d94792ffae..2b7e0604f838 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 2024
 PATCHLEVEL = 09
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20240918-1
 NAME = None
 
 # *DOCUMENTATION*
