From: =?UTF-8?q?Leonard=20G=C3=B6hrs?= <l.goehrs@pengutronix.de>
Date: Mon, 27 Mar 2023 14:57:33 +0200
Subject: [PATCH] Release 2023.03.0/customers/lxa/tac/20230327-2

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index ae44cbe003d0..e50e56c43306 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 2023
 PATCHLEVEL = 03
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20230327-2
 NAME = None
 
 # *DOCUMENTATION*
