From: =?UTF-8?q?Leonard=20G=C3=B6hrs?= <l.goehrs@pengutronix.de>
Date: Wed, 1 Mar 2023 11:22:41 +0100
Subject: [PATCH] ARM: stm32mp: lxa-tac: enable deep probe
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Leonard Göhrs <l.goehrs@pengutronix.de>
---
 arch/arm/boards/lxa-tac/board.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boards/lxa-tac/board.c b/arch/arm/boards/lxa-tac/board.c
index 6e7d4bf77635..0ebad2ec1de4 100644
--- a/arch/arm/boards/lxa-tac/board.c
+++ b/arch/arm/boards/lxa-tac/board.c
@@ -5,6 +5,7 @@
 #include <asm/memory.h>
 #include <mach/bbu.h>
 #include <bootsource.h>
+#include <deep-probe.h>
 #include <of.h>
 #include <tlv/tlv.h>
 #include <boards/lxa/tlv.h>
@@ -36,6 +37,7 @@ static const struct of_device_id tac_of_match[] = {
 	{ .compatible = "lxa,stm32mp157c-tac" },
 	{ /* sentinel */ },
 };
+BAREBOX_DEEP_PROBE_ENABLE(tac_of_match);
 
 static struct driver tac_board_driver = {
 	.name = "board-lxa-tac",
