From: =?UTF-8?q?Leonard=20G=C3=B6hrs?= <l.goehrs@pengutronix.de>
Date: Fri, 21 Apr 2023 07:50:33 +0200
Subject: [PATCH] Release 2023.03.0/customers/lxa/tac/20230421-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index ae44cbe003d0..78c865dda6c8 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 2023
 PATCHLEVEL = 03
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20230421-1
 NAME = None
 
 # *DOCUMENTATION*
