From: Philipp Zabel <p.zabel@pengutronix.de>
Date: Thu, 30 Mar 2023 15:19:14 +0200
Subject: [PATCH] Release 2023.03.0/customers/lxa/tac/20230330-4

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index ae44cbe003d0..5069a46a7329 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 2023
 PATCHLEVEL = 03
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20230330-4
 NAME = None
 
 # *DOCUMENTATION*
