From: Philipp Zabel <p.zabel@pengutronix.de>
Date: Fri, 31 Mar 2023 16:02:42 +0200
Subject: [PATCH] Release 2023.03.0/customers/lxa/tac/20230331-1

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index ae44cbe003d0..611988dc9e4d 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 2023
 PATCHLEVEL = 03
 SUBLEVEL = 0
-EXTRAVERSION =
+EXTRAVERSION =-20230331-1
 NAME = None
 
 # *DOCUMENTATION*
