From: Rouven Czerwinski <rouven@czerwinskis.de>
Date: Thu, 15 Jul 2021 09:14:58 +0200
Subject: [PATCH] lxatac: ETH PLL configuration

Signed-off-by: Rouven Czerwinski <rouven@czerwinskis.de>
---
 fdts/stm32mp157c-lxa-tac.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/fdts/stm32mp157c-lxa-tac.dts b/fdts/stm32mp157c-lxa-tac.dts
index c5f55cae1d2a..98286ac67aab 100644
--- a/fdts/stm32mp157c-lxa-tac.dts
+++ b/fdts/stm32mp157c-lxa-tac.dts
@@ -79,3 +79,8 @@
 		};
 	};
 };
+
+/* Configure Eth PLL 125Mhz Clock output */
+&pll4 {
+	cfg = <3 124 5 11 11 PQR(1,1,1)>;
+};
